AtReconstruction/AtPatternModificationTask.h

```diff
@@ -0,0 +1,49 @@
+#ifndef ATPATTERNMODIFICATIONTASK
+#define ATPATTERNMODIFICATIONTASK
+
+#include "AtPatternModification.h"
+
+#include <FairTask.h>
+
+#include <Rtypes.h>
+#include <TClonesArray.h>
+#include <TString.h>
+
+#include <memory>
+#include <utility>
+#include <vector>
+
+class AtPatternModificationTask : public FairTask {
```

Comment:
I think this needs to not do in-place modification. Or at least that should be an option and it only does in-place modification of the output and input branches are the same